lib/circuitry/configuration.rb

```diff
@@ -26,6 +30,14 @@ def subscribe_async_strategy=(value)
       super
     end
 
+    def subscriber_queue_name
+      [
+        application_name,
+        queue_prefix,
+        "events"
+      ].join('-')
+    end
```

Comment:
I'm hesitant to include this logic since it's specific to kapost's usage of circuitry, and not circuitry itself.